modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -1112,7 +1112,7 @@ protected function render_tab_titles_html( $item_settings ): string {
 					<?php echo $icon_active_html; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
 				</span>
 				<span <?php echo $text_class; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
-					<?php echo esc_html( $title ); ?>
+					<?php echo $title; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
```

Comment:
should we use here `wp_kses_post` with allowed html tags as an argument?

Reply:
Good suggestion. I have added it.